.all-contributorsrc

```diff
@@ -1,52 +1,84 @@
 {
-	"projectName": "utilities",
-	"projectOwner": "sapphire-project",
-	"repoType": "github",
-	"repoHost": "https://github.com",
-	"files": [
-		"README.md",
-		"packages/decorators/README.md",
-		"packages/eslint-config/README.md",
-		"packages/event-iterator/README.md",
-		"packages/ratelimits/README.md",
-		"packages/snowflake/README.md",
-		"packages/ts-config/README.md",
-		"packages/type/README.md",
-		"packages/utilities/README.md"
-	],
-	"imageSize": 100,
-	"commit": true,
-	"commitConvention": "angular",
-	"contributors": [
-		{
-			"login": "Favna",
-			"name": "Jeroen Claassens",
-			"avatar_url": "https://avatars3.githubusercontent.com/u/4019718?v=4",
-			"profile": "https://favware.tech/",
-			"contributions": ["code", "infra", "projectManagement", "doc", "test"]
-		},
-		{
-			"login": "kyranet",
-			"name": "Antonio Román",
-			"avatar_url": "https://avatars0.githubusercontent.com/u/24852502?v=4",
-			"profile": "https://github.com/kyranet",
-			"contributions": ["code", "projectManagement", "review", "test"]
-		},
-		{
-			"login": "PyroTechniac",
-			"name": "Gryffon Bellish",
-			"avatar_url": "https://avatars2.githubusercontent.com/u/39341355?v=4",
-			"profile": "https://github.com/PyroTechniac",
-			"contributions": ["code", "review", "test"]
-		},
-		{
-			"login": "vladfrangu",
-			"name": "Vlad Frangu",
-			"avatar_url": "https://avatars3.githubusercontent.com/u/17960496?v=4",
-			"profile": "https://github.com/vladfrangu",
-			"contributions": ["code", "bug", "review", "userTesting", "test"]
-		}
-	],
-	"contributorsPerLine": 7,
-	"skipCi": true
+  "projectName": "utilities",
+  "projectOwner": "sapphire-project",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md",
+    "packages/decorators/README.md",
+    "packages/eslint-config/README.md",
+    "packages/event-iterator/README.md",
+    "packages/ratelimits/README.md",
+    "packages/snowflake/README.md",
+    "packages/ts-config/README.md",
+    "packages/type/README.md",
+    "packages/utilities/README.md"
```

Comment:
Add `packages/time-utilities/README.md` to this and then run `all-contributors generate` (using the all contributors CLI)

Reply:
Done